errors.go

```diff
@@ -93,11 +93,24 @@ type ValidationError struct {
 }
 
 func (e ValidationError) Error() string {
+	var (
+		location string
+		context  string
+	)
+
 	if e.Field == "" {
-		return fmt.Sprintf("line %d: %s", e.Line, e.Err)
+		location = fmt.Sprintf("line %d", e.Line)
+	} else {
+		location = fmt.Sprintf("line %d, field %s", e.Line, e.Field)
+	}
+
+	if e.Context != nil {
+		context = fmt.Sprintf("%v", e.Context)
+	} else {
+		context = "no context provided"
```

Comment:
If a context isn't supplied, I presume there is none available. I am more in favor of it not providing a message if no context is available.